cmd/wallet-web/src/pages/chapi/Login.vue

```diff
@@ -36,13 +39,16 @@ SPDX-License-Identifier: Apache-2.0
 <script>
     import {DeviceLogin, RegisterWallet} from "./wallet"
     import {mapActions, mapGetters} from 'vuex'
+    import Vue from 'vue';
 
     export default {
         created: async function () {
+           this.registerSuccess = Vue.$cookies.get("device");
```

Comment:
Cookies should not be set/read by the frontend :(.

Reply:
This cookie dont have sensitive data it is just to enable / disable  login with touch id button in the front end. Now reading this value only enabling Sign in with touch id option if the device has been registered. I can add todo to re-visit, but in my opinion it should be fine.

Reply:
@talwinder50 if the user wipes their browser's storage (re-install, etc) then the "Login With Touch ID" button would not be enabled, correct?

This would be better implemented with information from the backend.

I guess that's a TODO.

Reply:
add #601 as todo